pimd/pim_mroute.c

```diff
@@ -61,11 +61,7 @@ static int pim_mroute_set(int fd, int enable)
     return -1;
   }
 
-  err = setsockopt(fd, SOL_SOCKET, SO_RCVBUF, &rcvbuf, sizeof(rcvbuf));
-  if (err) {
-    zlog_warn("%s: failure: setsockopt(fd=%d, SOL_SOCKET, %d): errno=%d: %s",
-	      __PRETTY_FUNCTION__, fd, rcvbuf, errno, safe_strerror(errno));
-  }
+  setsockopt_so_recvbuf(fd, 1024 * 1024 * 8);
```

Comment:
I know we are doing this in multiple places, but I wonder if the size should be a symbol/constant so that It could be easily changed across all sockets